PLUGINS_CATALOG.md

```diff
@@ -0,0 +1,37 @@
+# Plugin Catalog
```

Comment:
The resulting file in the Snap repo is `PLUGIN_CATALOG.md`. Would make more sense for this name to be consistent.